container/src/container/array_dict.py

```diff
@@ -9,196 +9,194 @@ def isiterable(obj):
         return False
 
 class array_dict:
```

Comment:
Oups.
Je suis allé vite et j'ai vu des erreurs.
Je pense néanmoins que ça ferais du bien au soft si on faisait plus de
code review.
Ca te tente?
A+
Christophe

PS: Quel est l'objet de ta PR?


Le 26/05/2016 16:21, Jonathan LEGRAND a écrit :
>
> In container/src/container/array_dict.py
> <https://github.com/openalea/openalea-components/pull/26#discussion_r64753938>:
>
> > @@ -9,196 +9,194 @@ def isiterable(obj):
> >          return False
> >  
> >  class array_dict:
>
> I did not changed anything in the 'array_dict' class ... as *not*
> showed by the whole diff, it seems that the differences are tabs
> converted into spaces...
> The name was like this before my commit...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/openalea/openalea-components/pull/26/files/719a5228f3c324afd26585802271969672232843#r64753938>
>